Sources/Public/Item Views/SwiftUIWrapperView.swift

```diff
@@ -0,0 +1,198 @@
+    guard let closestViewController = closestViewController() else {
+      assertionFailure(
+        "Could not find a view controller to which the `UIHostingController` could be added.")
+      return
+    }
+
+    guard hostingController.parent !== closestViewController else { return }
+
+    if hostingController.parent != nil {
+      hostingController.willMove(toParent: nil)
+      hostingController.view.removeFromSuperview()
+      hostingController.removeFromParent()
+      hostingController.didMove(toParent: nil)
+    }
+
+    hostingController.willMove(toParent: closestViewController)
+    closestViewController.addChild(hostingController)
+    hostingControllerView = hostingController.view
+    addSubview(hostingController.view)
+    hostingController.didMove(toParent: closestViewController)
+
+    setNeedsLayout()
+  }
+
+  // This allows touches to be passed to `ItemView` even if the SwiftUI `View` has a gesture
+  // recognizer.
+  private func configureGestureRecognizers() {
+    for gestureRecognizer in hostingControllerView?.gestureRecognizers ?? [] {
+      gestureRecognizer.cancelsTouchesInView = false
+    }
+  }
+
+}
+
+// MARK: CalendarItemViewRepresentable
+
+@available(iOS 13.0, *)
+extension SwiftUIWrapperView: CalendarItemViewRepresentable {
+
+  public struct InvariantViewProperties: Hashable {
+
+    // MARK: Lifecycle
+
+    init(initialContent: Content) {
+      self.initialContent = initialContent
+    }
+
+    // MARK: Public
+
+    public static func == (lhs: InvariantViewProperties, rhs: InvariantViewProperties) -> Bool {
+      type(of: lhs.initialContent) == type(of: rhs.initialContent)
+    }
+
+    public func hash(into hasher: inout Hasher) { }
+
+    // MARK: Fileprivate
+
+    fileprivate let initialContent: Content
+
+  }
+
+  public struct ViewModel: Equatable {
+
+    // MARK: Lifecycle
+
+    public init(content: Content) {
+      self.content = content
+    }
+
+    // MARK: Public
+
+    public static func == (_: ViewModel, _: ViewModel) -> Bool {
+      false
+    }
+
+    // MARK: Fileprivate
+
+    fileprivate let content: Content
+
+  }
+
+  public static func makeView(
+    withInvariantViewProperties invariantViewProperties: InvariantViewProperties)
+    -> SwiftUIWrapperView<Content>
+  {
+    SwiftUIWrapperView<Content>(content: invariantViewProperties.initialContent)
+  }
+
+  public static func setViewModel(_ viewModel: ViewModel, on view: SwiftUIWrapperView<Content>) {
+    view.content = viewModel.content
+  }
+
+}
+
+// MARK: UIResponder Next View Controller Helper
+
+extension UIResponder {
+  /// Recursively traverses up the responder chain to find the closest view controller.
+  fileprivate func closestViewController() -> UIViewController? {
+    self as? UIViewController ?? next?.closestViewController()
+  }
+}
+
+// MARK: - HostingController
+
+@available(iOS 13.0, *)
+private final class HostingController<Content: View>: UIHostingController<Content> {
```

Comment:
It might make sense to make this type public or give it a more recognizable name so that it can easily be excluded from UIViewController-based viewDidAppear swizzling impression systems, just in case anyone has something like that